.circleci/config.yml

```diff
@@ -135,6 +163,10 @@ jobs:
       - store_test_results:
           path: ~/junit/
           destination: testartifacts
+      - run:
```

Comment:
output the json data that circleci uses to compute the times for tests, this is useful when debugging.